common/djangoapps/student/helpers.py

```diff
@@ -226,7 +226,7 @@ def check_verify_status_by_course(user, course_enrollments):
 POST_AUTH_PARAMS = ('course_id', 'enrollment_action', 'course_mode', 'email_opt_in', 'purchase_workflow')
 
 
-def get_next_url_for_login_page(request):
+def get_next_url_for_login_page(request, add_host_in_redirect=False):
```

Comment:
nit: Rename the keyword argument.
```suggestion
def get_next_url_for_login_page(request, include_host=False):
```